typescript-react/src/components/validation/type.ts

```diff
@@ -12,7 +13,7 @@ import { validatorCreatorFactory } from './validatorCreatorFactory';
 export const isIntegerCreator =
   validatorCreatorFactory<void, Numeric, any, any>({
     getValidatorErrorMessage: (_, input) => !(Number(input) <= INT_MAX_VALUE && Number(input) >= INT_MIN_VALUE) ?
-      'Must be a whole number with up to 10 digits' :
+      `Must be a whole number between ${formatNumberToDecimals(INT_MIN_VALUE, 0)} and ${formatNumberToDecimals(INT_MAX_VALUE, 0)}` :
```

Comment:
Shouldn't this be some other constant instead of being calculated here every time?

Reply:
Sure, changed

Reply:
I meant the whole message :)

Reply:
I mean, its most likely not important :D
It just stands out to me when I read it

Reply:
Maybe this time you'll be happy :D

Reply:
Also, squashed the commits

Reply:
:)
When I look at that code the behavior I prefer is to have that message string as constant created only once and only when needed
This code does create it only once, but I dont know if it created only when needed.
I mean its probably not worth it, but when you ignore such reasoning all the time you end up bleeding from 1000 cuts